src/components/ManageDataModal.js

```diff
@@ -279,7 +279,10 @@ function ResetDataModalContent({ setIsOpen }) {
                     type='button'
                     onClick={() => {
                         resetLocalStorageData();
-                        setSuccessText('Data has been deleted.');
+                        setSuccessText('Data has been deleted. Reloading page...');
+                        setTimeout(() => {
+                            window.location.reload();
+                        }, 1000);
```

Comment:
Oops i left this in - but given a page refresh is needed after the data has been synced let me know if you like this or not. Can always remove.

Reply:
All good - this is probably good to have anyway